acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsDefaultImpl.java

```diff
@@ -212,7 +224,48 @@ private Collection<String> createEth2Args() {
     return params;
   }
 
-  private String createCommaSeparatedList(final List<String> values) {
+  private String createCommaSeparatedList(final Collection<String> values) {
     return String.join(",", values);
   }
+
+  private Collection<String> awsCliOptions() {
+    final List<String> params = new ArrayList<>();
+
+    if (signerConfig.getAwsSecretsManagerParameters().isPresent()) {
```

Comment:
Similar to comment in CmdLineParamsConfigFileImpl think would be nicer to have the if in the layer above

Reply:
done.